### ISSUE TITLE
undefined symbol: _ZTI12QCPFinancial

### ISSUE THREAD

**Issue by altendky** [dimV36/QCustomPlot-PyQt5#1]
Hi Dimitry,

When I try to `import qcustomplot` i get the error `undefined symbol: _ZTI12QCPFinancial`.  I did make some adjustments to `configure.py` (f65011788eb54e4e7caec156de8edf4add7ba397) to get the build working on Kubuntu 15.04 with Python3.  I also symlinked from `libqcustomplotlib.so` to `libqcustomplotlib.so.1.2`.  I'll keep digging but any chance you have had such an issue when working on this?

You can see my boring build log below.

https://gist.github.com/altendky/2a2c78d63ae0a2b5f7f7

Cheers, (and thanks for sharing this)
-kyle

**Comment by altendky** [comment]
:[ Nevermind.  As soon as I typed the `1.2` above I realized it.  The financial stuff was introduced in 1.3.0.  I had 1.2 because that was what was available in the Kubuntu repos.  Downloading and building the latest shared library made it work fine.

Sorry for the interruption...

**Comment by altendky** [comment]
Or maybe not.  Heading to bed but the next issues is `undefined symbol: _ZNK11QCustomPlot15plottableBarsAtERK7QPointFb`.

`$ c++filt _ZNK11QCustomPlot15plottableBarsAtERK7QPointFb
QCustomPlot::plottableBarsAt(QPointF const&, bool) const`

What version have you built QCustomPlot-PyQt5 against?

Thanks.

**Comment by altendky** [comment]
Eventually I realized that your repo has a modified version of QCustomPlot 1.3.1 defining a couple extra functions.  Once I reverted this (c2795dadbbce47b9801eae27c36b7341a8b17b00) to match the git checkout of QCustomPlot I used to build the libraries I was able to `import qcustomplot` without errors.  With this I successfully created a plot window with data.  Now to learn QCustomPlot itself.

Thanks again.
